pkg/proxy/winkernel/proxier_test.go

```diff
@@ -20,7 +20,7 @@ package winkernel
 
 import (
 	"k8s.io/api/core/v1"
-	discovery "k8s.io/api/discovery/v1alpha1"
```

Comment:
At what stage do we add EPSlice support to Windows proxiers?

Reply:
Hopefully soon, maybe 1.18. I'll try to reach out to some of the people who are more familiar with that side of things to see if we can work together to get EndpointSlice integration in place.